src/react/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  createContext,
+  createElement,
+  useCallback,
+  useContext,
+  useEffect,
+  useImperativeHandle,
+  useLayoutEffect,
+  useMemo,
+  useRef,
+  useState,
```

Comment:
Ok, I figured out my issue on my branch.  TLDR I had to add a lot of `import, export, import, export` for various features.

The good news is, after re-exporting all these features, it works!  
The bad news is, it took a lot of manual hunt-and-peck to find all the issues, and the process was pretty tedious.  
I'm going to experiment and see if I can do something along the lines of `export * from '#preact'`

Reply:
This works for me: replace this whole file with:
```
import * as Preact from '#preact';
import * as PreactCompat from '#preact/compat';

export * from '#preact';
export * from '#preact/compat';

// This file allows us to remap react imports from external libraries to
// our internal preact exports. This file uses a default export in order to
// be compatible with libraries that use react default import syntax.
export default {...Preact, ...PreactCompat};
```

With this `*` in place, I only need to update `/src/preact/index.js` whenever a 3rd party lib tries to use a new API (eg. `useDebugValue` or `Component`)

Reply:
Sounds good, I will make this update!